README.md

```diff
@@ -1,26 +1,57 @@
-# Offscreen Effect Player module
+# [Offscreen Effect Player module (OEP-module)](https://docs.banuba.com/face-ar-sdk-v1/core/cpp_api_overview#offscreen-effect-player)
+
+[Description of the **Offscreen Effect Player** on the Banuba SDK website](https://docs.banuba.com/face-ar-sdk-v1/core/cpp_api_overview#offscreen-effect-player)
 
 ## Description
 
-Common submodule for some projects using **offscreen_effect_player**
-The module implements interaction with the effect player. The module consists of six main interfaces:
+Common submodule for some projects using [**Offscreen effect player**](./interfaces/offscreen_effect_player.hpp). The module implements interaction with the effect player.
+
+## Folders structure
+
+- [**docs**](./docs/) - illustrations used in this README.md
+- [**interfaces**](./interfaces/) - offscreen effect player interfaces
+- [**offscreen_effect_player**](./offscreen_effect_player/) - is a wrapper for effect_player. It allows you to use your own implementation for offscreen_render_target
+- [**offscreen_render_target**](./offscreen_render_target/) - is an implementation option for the offscreen_render_target interface. Allows to prepare gl framebuffers and textures for receiving a frame from gpu, receive bytes of the processed frame from the gpu and pass them to the cpu, as well as, if necessary, set the orientation for the received frame. This implementation uses GLFW to work with gl context
+- [**opengl**](./opengl/) - OpenGL utilities
+- [**third**](./third/) - third party libraries
+
+## Description of interfaces
+
+The module consists of six main interfaces.
+- [**effect_player**](./interfaces/effect_player.hpp) - implementation contains interaction with the effect player. Should be implemented on the application side.
```

Comment:
Why are you talking about implementation when you need to describe the interface?